src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/CaseSearchQuerySecurity.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.ccd.domain.service.search.elasticsearch;
```

Comment:
I think we should organise better the package structure. I would move any class that deals with security under a .security subpackage

Reply:
done